crates/cadenza-syntax/src/generated/snapshots/cadenza_syntax__generated__test_data__fn_with_block__fn_with_block_ast.snap

```diff
@@ -0,0 +1,7 @@
+---
+source: crates/cadenza-syntax/src/generated/test_data.rs
+expression: "fn compute x =\n  let a = x * 2\n  let b = a + 10\n  b / 2\n"
+---
+[
+    [=, [[fn, compute], x], [[=, [[[=, [let, a], [*, x, 2]], let], b], [+, a, 10]], [/, b, 2]]],
```

Comment:
just looking at this, i think the AST really needs to be structured with an implicit `[__block__, ...exprs]` to make it easy to process. where parent nodes can just treat it as any old apply expression that can be evaluated.